stl/src/msvcp_atomic_wait.def

```diff
@@ -0,0 +1,16 @@
+; Copyright (c) Microsoft Corporation.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; atomic wait satellite DLL definition
+
+
+EXPORTS

```

Comment:
Should these be lexicographically sorted?

Reply:
Why?
Exports from binary will be sorted anyway.
In source code the current order is more logical, lexicographical will give "unwait" above "wait" and inconvenient grouping